build.bash

```diff
@@ -164,10 +221,10 @@ grow_image() {
   local partition
 
   # root partition is #2 and sector size is 512 byte for the Raspi OS image
-  partition=2
+  partition="2"
 
   dd if=/dev/zero bs=1M count="$2" oflag=append conv=notrunc of="$1" &> /dev/null
-  partStart=$(echo p|/sbin/fdisk "$1" | grep "$1"2 | awk '{print $2}')
+  partStart="$(echo p|/sbin/fdisk "$1" | grep "$1"2 | awk '{print $2}')"
```

Comment:
any recent changes here ? is partStart still computed correctly ?

Reply:
I believe so, I have not seen any errors in that.